setup.py

```diff
@@ -111,8 +111,6 @@ def get_long_description():
           url='http://avocado-framework.github.io/',
           use_2to3=True,
           packages=find_packages(exclude=('selftests*',)),
```

Comment:
At first I was afraid this would include the `optional_plugins`, but it ignores it...